README.md

```diff
@@ -37,17 +37,17 @@ opposed to agent-based monitoring, where the monitoring code runs in the same lo
 as the endpoint).
 
 For an example walkthrough of creating a new Management Pack monitoring an endpoint, see
-[Creating a new Management Pack](https://vmware.github.io/vmware-aria-operations-integration-sdk/get_started/#creating-a-new-management-pack)
+[Creating a new Management Pack](https://vmware.github.io/vmware-aria-operations-integration-sdk/get_started/#creating-a-management-pack)
```

Comment:
It'd be better to rename the section "Creating a New Management Pack", to differentiate it from "Extending an Existing Management Pack".